### PR TITLE
Merge Hdrp/staging

### PR DESCRIPTION
**Pull request by sebastienlagarde**
### Purpose of this PR
- Introducing shadow resolution quality tiers (#3988) 
- Hdrp/additional velocity changes support for alembic (#3981)
- Moved HDRP/Core to Core (#3989)
- Update Reflection settings +Rename EnableSpecularLighting (#3986)
- Add an option to strip terrain hole shader variant (#3937)
- refactored lightloop and added XrMaxViews to ShaderConfig (#3950)
- fixed indirect deferred compute with XR single-pass instancing (#3991)
- Fix remaining ray tracing shader warnings (#4003)
- fix compil issue after merge
- Refactor Deferred Lighting to be compatible with render graph (#3975)  …
- Delete IndirectDiffuseAccumulation.compute.meta
- Hdrp/alembic motionvectors shadergraph patch (#3996)

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
- [ ] Opened test project + Run graphic tests locally
- [ ] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=HDRP%2Fstaging&automation-tools_branch=master&unity_branch=trunk&sort=1-asc

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?

**Halo Effect**: None, Low, Medium, High?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Comment by sebastienlagarde** [comment]
Katana is green

File: TestProjects/UniversalGraphicsTest/Assets/Plugins.meta [hunk 1]
```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```
Comment (phi-lira) [inline TestProjects/UniversalGraphicsTest/Assets/Plugins.meta]: This metafile move seems weird. Says Common.meta inside HDRP moved to TestProjects?

File: com.unity.render-pipelines.core/Editor/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+using System.Runtime.CompilerServices;
```
Comment (phi-lira) [inline com.unity.render-pipelines.core/Editor/AssemblyInfo.cs]: Please remove this file. Doesn't add anything to project. Also the respective meta should be deleted.

Comment (sebastienlagarde) [reply com.unity.render-pipelines.core/Editor/AssemblyInfo.cs]: Done

**Pull request merged**